src/components/settings/SettingsLMSTab/index.jsx

```diff
@@ -63,12 +68,13 @@ const SettingsLMSTab = ({
     const options = { enterprise_customer: enterpriseId };
     LmsApiService.fetchEnterpriseCustomerIntegrationConfigs(options)
       .then((response) => {
-        setShowNewConfigButtons(true);
+        setShowNoConfigCard(true);
```

Comment:
Would a better place for this be the else clause on the response.data.length check below?